src/components/React/VolumeRenderer.js

```diff
@@ -79,23 +72,55 @@ export default connect(mapStateToProps)
             cursor-listener 
           /> 
 
-        
-          <Entity 
-            id="volumeCube" 
-            class="clickableMesh"   
-            myloader={{
-              volumeData:this.props.volumeData,rayCollided:false,
-              transferFunction:this.props.transferFunction,colorMap:this.props.colorMap,
-              opacity1:this.props.opacity1,opacity2:this.props.opacity2,
-              lowNode:this.props.lowNode,highNode:this.props.highNode,
-              alphaXDataArray:this.props.alphaXDataArray,
-              alphaYDataArray:this.props.alphaYDataArray,
-              colorMapping:this.props.colorMapping,
-              channel:this.props.channel,
-              cameraState:this.props.cameraState
-            }}   
-            position="0 0 0"
-          />
+          <VolumeConsumer>
+            {({state}) => (
+              <Entity 
+                id="volumeCube" 
+                class="clickableMesh"   
+                myloader={{
+                  volumeData: state.path,
+                  rayCollided: false,
+                  transferFunction: this.props.transferFunction,
+                  colorMap: this.props.colorMap,
+                  opacity1: this.props.opacity1,
+                  opacity2: this.props.opacity2,
+                  lowNode: this.props.lowNode, 
+                  highNode:this.props.highNode,
+                  alphaXDataArray: this.props.alphaXDataArray,
+                  alphaYDataArray: this.props.alphaYDataArray,
+                  colorMapping: this.props.colorMapping,
+                  channel: this.props.channel,
+                  cameraState: this.props.cameraState
+                }}   
+                position="0 0 0"
+              />
+            )}
+          </VolumeConsumer>
+
+          {/* <VolumeContext.Consumer>
```

Comment:
remove commented out code (or add a comment on why it's commented out for now, but when x happens it'll be used)

Reply:
Whoops, good catch. Just deleted it. Some of it's been commented since before I joined. I've left that as is, should I just go ahead and delete it?

Reply:
delete - it's in git if we ever need to go look at it

Reply:
okay!